P3.txt

```diff
@@ -0,0 +1,16 @@
+P3. Compute the sum of n numbers
+
+val n: Int
+read(n)
+
+if (n <= 0)
+	write(“Can’t be done”)
+else {
+	val arr: Int[n]
+	for (0 to n - 1)

```

Comment:
As mentioned in the P1.txt for cannot have as parameter an expression (e.g., n - 1).